ci-operator/step-registry/upi/deprovision/gcp/upi-deprovision-gcp-commands.sh

```diff
@@ -8,9 +8,61 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 
 export HOME=/tmp
 
+# release-controller always expose RELEASE_IMAGE_LATEST when job configuraiton defines release:latest image
+echo "RELEASE_IMAGE_LATEST: ${RELEASE_IMAGE_LATEST:-}"
+# seem like release-controller does not expose RELEASE_IMAGE_INITIAL, even job configuraiton defines 
+# release:initial image, once that, use 'oc get istag release:inital' to workaround it.
+echo "RELEASE_IMAGE_INITIAL: ${RELEASE_IMAGE_INITIAL:-}"
+if [[ -n ${RELEASE_IMAGE_INITIAL:-} ]]; then
+    tmp_release_image_initial=${RELEASE_IMAGE_INITIAL}
+    echo "Getting inital release image from RELEASE_IMAGE_INITIAL..."
+elif oc get istag "release:initial" -n ${NAMESPACE} &>/dev/null; then
+    tmp_release_image_initial=$(oc -n ${NAMESPACE} get istag "release:initial" -o jsonpath='{.tag.from.name}')
+    echo "Getting inital release image from build farm imagestream: ${tmp_release_image_initial}"
+fi
+# For some ci upgrade job (stable N -> nightly N+1), RELEASE_IMAGE_INITIAL and 
+# RELEASE_IMAGE_LATEST are pointed to different imgaes, RELEASE_IMAGE_INITIAL has 
+# higher priority than RELEASE_IMAGE_LATEST
+TESTING_RELEASE_IMAGE=""
+if [[ -n ${tmp_release_image_initial:-} ]]; then
+    TESTING_RELEASE_IMAGE=${tmp_release_image_initial}
+else
+    TESTING_RELEASE_IMAGE=${RELEASE_IMAGE_LATEST}
+fi
+echo "TESTING_RELEASE_IMAGE: ${TESTING_RELEASE_IMAGE}"
+
+# check if OCP version will be equal to or greater than the minimum version
+# $1 - the minimum version to be compared with
+# return 0 if OCP version >= the minimum version, otherwise 1
+function version_check() {
+  local -r minimum_version="$1"
+
+  dir=$(mktemp -d)
+  pushd "${dir}"
+
+  cp ${CLUSTER_PROFILE_DIR}/pull-secret pull-secret
+  oc registry login --to pull-secret
```

Comment:
To be safe, suggesting
```suggestion
  KUBECONFIG="" oc registry login --to pull-secret
```